From 2e1dd622ee7020a608c3397768ea245dcb6409d1 Mon Sep 17 00:00:00 2001
From: David Herrmann <dh.herrmann@gmail.com>
Date: Tue, 2 Sep 2014 14:19:38 +0200
Subject: [PATCH] terminal: remove redundant "struct" prefixes

We define typedefs for all internal types so drop the redundant "struct"
prefix.
---
 src/libsystemd-terminal/idev-evdev.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/libsystemd-terminal/idev-evdev.c b/src/libsystemd-terminal/idev-evdev.c
index 9e2dc811ef..be9b0301a4 100644
--- a/src/libsystemd-terminal/idev-evdev.c
+++ b/src/libsystemd-terminal/idev-evdev.c
@@ -41,7 +41,7 @@ typedef struct unmanaged_evdev unmanaged_evdev;
 typedef struct managed_evdev managed_evdev;
 
 struct idev_evdev {
-        struct idev_element element;
+        idev_element element;
         struct libevdev *evdev;
         int fd;
         sd_event_source *fd_src;
@@ -52,12 +52,12 @@ struct idev_evdev {
 };
 
 struct unmanaged_evdev {
-        struct idev_evdev evdev;
+        idev_evdev evdev;
         char *devnode;
 };
 
 struct managed_evdev {
-        struct idev_evdev evdev;
+        idev_evdev evdev;
         dev_t devnum;
 
         sd_bus_slot *slot_pause_device;
